WPiOS Integration.md

```diff
@@ -0,0 +1,51 @@
+# WordPress iOS Integration Guide
+
+## How to work with local gutenberg-mobile checkout
+
+You need to set `LOCAL_GUTENBERG` environment variable to the local `gutenberg-mobile` folder relative to the `WPiOS` folder.  
```

Comment:
👋 Ceyhun, thanks for opening this PR!

I wonder, what do you think about adding a small "intro" paragraph about where `LOCAL_GUTENBERG` plays a role? I mean, it looks like the only effect the flag has is to set cocoapods up for using a locally available checkout of the gutenberg-mobile repo, from which then the App JS bundle is picked up, right? And for example, it plays no particular role to which WPiOS run scheme will be selected, whether Metro will be used or not, etc. I feel a short paragraph like that to set the overall stage about what `LOCAL_GUTENBERG` is about can be useful. wdyt?

Reply:
Sounds good 👍